pkg/infoschema/test/clustertablestest/cluster_tables_test.go

```diff
@@ -1228,65 +1228,6 @@ func TestUniversalBindingFromHistory(t *testing.T) {
 	tk.MustQuery(`select @@last_plan_from_binding`).Check(testkit.Rows("1"))
 }
 
-func TestStmtSummaryShowPlanForSQL(t *testing.T) {
```

Comment:
Remove these 2 unstable tests.

Reply:
any replacements for those 2 cases? to make sure the path tested by them are still covered

Reply:
Not yet, actually the syntax "show plan for" will be even removed later on.
Now the new syntax is "explain explore" and I think its current test cases (and future cases) can cover these two's.

Reply:
lgtm